lib/lighthouse/facilities/facility.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+# Facility Model
+module Lighthouse
+  module Facilities
+    class Facility < Common::Base
+      attribute :id, String
+      attribute :type, String
+      attribute :name, String
+      attribute :facility_type, String
+      attribute :classification, String
+      attribute :website, String
+      attribute :lat, Float
+      attribute :long, Float
+      attribute :address, Object
+      attribute :phone, Object
+      attribute :hours, Object
+      attribute :services, Object
+      attribute :feedback, Object
+      attribute :access, Object
+      attribute :mobile, Boolean
+      attribute :active_status, String
+      attribute :visn, String
+      attribute :operating_status, Object
+      attribute :facility_type_prefix, String
+      attribute :unique_id, String
+
+      def initialize(fac)
```

Comment:
A test that this init is inserting all of the attributes correctly might be helpful.

Reply:
The tests for `get_by_id` and `get_facilities` both use 
`expect(r).to have_attributes(vha_358_attributes)` 
to verify the attributes of the returned object. Are you thinking of something more than that like tests for the model itself?

Reply:
this needs to call super

Reply:
I really wish the serializers lived in the serializer directory.

Reply:
Serializer directory? Do you mean you wish this was in app/models?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/tree/master/app/serializers

Reply:
I'm confused. I added this model so that the response is serializable, but am assuming serialization will take place at the controller. Am I making this model do too much?

Reply:
ok, it looked like you were following the active support serializer format.